src/patternfly/components/Touchspin/touchspin.scss

```diff
@@ -0,0 +1,55 @@
+.pf-c-touchspin {
+  // main
+  --pf-c-touchspin__main--m-active--c-form-control--Color: var(--pf-global--Color--100);
+
+  // units
+  --pf-c-touchspin__main__units--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-touchspin__units__main--MarginLeft: var(--pf-global--spacer--sm);
+
+  // control
+  --pf-c-touchspin__control--m-threshold--c-button--Color: var(--pf-global--disabled-color--100);
+
+  // icon
+  --pf-c-touchspin__control-icon--FontSize: var(--pf-global--FontSize--xs);
+
+  // form control
+  --pf-c-touchspin--c-form-control--Color: var(--pf-global--disabled-color--100);
+  --pf-c-touchspin--c-form-control--MaxWidth: 6ch;
+
+  display: flex;
+  align-items: center;
+
+  .pf-c-form-control {
+    display: inline;
+    max-width: var(--pf-c-touchspin--c-form-control--MaxWidth);
+    color: var(--pf-c-touchspin--c-form-control--Color);
+    text-align: right;
+  }
+}
+
+.pf-c-touchspin__main {
```

Comment:
Do you anticipate needing the `__main` element? I was assuming the grey/black text change would come from placholder/regular value text, and even if we kept the `pf-m-active` modifier, it could apply to the parent `pf-c-touchspin` component class, too.